src/Utility/Security.php

```diff
@@ -121,17 +121,35 @@ public static function randomBytes(int $length): string
      * Creates a secure random string.
      *
      * @param int $length String length. Default 64.
+     * @param bool $inclSpecialChars If include special characters. Default false.
      * @return string
      */
-    public static function randomString(int $length = 64): string
+    public static function randomString(int $length = 64, bool $inclSpecialChars = false): string
```

Comment:
```suggestion
    public static function randomString(int $length = 64, bool $includeSpecialChars = false): string
```

With PHP supporting named parameters, lets not introduce word fragments.

Reply:
Updated $inclSpecialChars, now is $includeSpecialChars (also in comment and inside function)

Reply:
I think it might be better if we made the parameter the special characters instead of hard-coding it.

Reply:
Good point. I've gave it some thought and redone the whole function. Allowing to custom character sets or using predefined lowercase, uppercase, special and numbers.